File: moi-dotnet/MoiBackend.Core/DTOs/GoldEntryResponse.cs
namespace MoiBackend.Core.DTOs;

public class GoldEntryResponse
{
    public long Id { get; set; }
    public long SerialNumber { get; set; }
    public string ContributorName { get; set; } = string.Empty;
    public string Village { get; set; } = string.Empty;
    public string GoldDetails { get; set; } = string.Empty;
    public DateTime EntryDate { get; set; }
    public long EventId { get; set; }
}

File: moi/src/main/java/com/moi/dto/GoldEntryResponse.java
package com.moi.dto;

import lombok.Builder;
import lombok.Data;
import java.time.LocalDateTime;

@Data
@Builder
public class GoldEntryResponse {
    private Long id;
    private Long serialNumber;
    private String contributorName;
    private String village;
    private String goldDetails;
    private LocalDateTime entryDate;
    private Long eventId;
}
